packages/ui-tree-browser/src/TreeBrowser/TreeCollection/index.js

```diff
@@ -187,7 +189,9 @@ class TreeCollection extends Component {
         numChildren={childCount}
         level={this.props.level + 1}
         position={i + 1}
-      />
+      >
+        {collection.children}
```

Comment:
Why is the property plural as "children", if the description says "allow tree browser collections to render a child node"? I also see the property as a single value:

`children: PropTypes.oneOfType([PropTypes.node, PropTypes.func])`

Reply:
I was referring to children as the React term of children as in [here]( https://learn.co/lessons/react-this-props-children). 

I don't have a huge preference on naming. Calling it "child" seems vague (maybe children is too if thinking outside of react), so I was also thinking of calling it childNode or childElement.

Reply:
I'm fine with plural, but the propTypes only accepts a single value (unless we would pass multiple children within a React Fragment), which was confusing me (I was looking for the propTypes for children, and I could only find [this](https://stackoverflow.com/a/42122662)).

Reply:
discussed a bit in slack, but went with PropType.node rather than something like:

```static propTypes = {
    children: PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.node),
        PropTypes.node
    ]).isRequired
``` 


because of what instui already uses